src/sagemaker_training/modules.py

```diff
@@ -129,6 +129,23 @@ def install(path, capture_error=False):  # type: (str, bool) -> None
     )
 
 
+def install_requirements(path, capture_error=False):  # type: (str, bool) -> None
+    """Install dependencies from requirements.txt in the executing Python environment.
+
+    Args:
+        path (str):  Real path location of the requirements.txt file.
+        capture_error (bool): Default false. If True, the running process captures the
+            stderr, and appends it to the returned Exception message in case of errors.
+    """
+    cmd = "%s -m pip install -r requirements.txt" % process.python_executable()
```

Comment:
favor `.format`?

Reply:
or f-string?
```
md = f"{process.python_executable()} -m pip install -r requirements.txt"
```

Reply:
We unfortunately still have to support Python 2 for now, so f-strings aren't on the table yet. Once we drop Python 2 support, we can switch to f-strings.